### DISCUSSION TITLE
zh/docs/provider/oauth/Wechat/

### DISCUSSION THREAD

**Discussion by giscus** [casdoor/casdoor#2893] [Docs comments]
# zh/docs/provider/oauth/Wechat/

将微信 OAuth 提供商添加到您的应用程序

https://casdoor.org/zh/docs/provider/oauth/Wechat/

**Comment by clhey** [comment]
商业系统authing可以支持微信移动端登录，https://docs.authing.cn/v2/guides/connections/social/wechat-mobile/ 。casdoor这边有考虑支持么？
